src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue

```diff
@@ -92,6 +116,123 @@ export default class DOWLandingPage extends Vue {
+        ((currEnv.statement_replicated_optimized !== "") &&
+            ((currEnv.additional_growth === 'NO') || (currEnv.additional_growth === 'YES' &&
+                (currEnv.anticipated_yearly_additional_capacity as unknown as string) !== "")) &&
+            ((currEnv.has_phased_approach === "NO") ||
+                (currEnv.has_phased_approach === 'YES' && currEnv.phased_approach_schedule !== "")))
+    // label
+    let label = null;
+    if (currEnv.current_environment_replicated_optimized === "YES_REPLICATE") {
+      label = "Replicate (lift and shift) using JWCC offerings";
+    } else if (currEnv.current_environment_replicated_optimized === "YES_OPTIMIZE") {
+      label = "Optimize (improve/modernize) using JWCC offerings";
+    } else if (currEnv.current_environment_replicated_optimized === "NO") {
+      label = "No requirements";
+    }
+    if (label !== null) { // only set the label if the user had selected one of the options
+      this.requirementSections[reqSectionIndex].label = label;
+    }
+  }
+
+  /**
+   * Sets the 'label' and 'isComplete' properties of the architectural section.
+   *
+   * Label gets built using the classification level options that the user selects. Default label
+   * is left as is, if no levels are selected
+   *
+   * Architectural design solution is complete, if all the below are true;
+   * 1. If the user does not need an architectural solution.
+   * 2. If the user needs an architectural solution AND
+   *      provides a detailed statement.
+   * 3. If the user needs an architectural solution AND
+   *      identifies at least one classification level that needs a solution.
+   */
+  async setArchitecturalDesignProperties(): Promise<void> {
+    const currEnv = CurrentEnvironment.currentEnvironment;
+    const currEnvArchNeeds = await CurrentEnvironment.getCurrentEnvironmentArchitecturalNeeds();
+    // is complete check
+    const reqSectionIndex =this.requirementSections
+      .findIndex(reqSection => reqSection.section === "ArchitecturalDesign")
+    this.requirementSections[reqSectionIndex].isComplete =
+        (currEnv.needs_architectural_design_services === "NO") ||
+        ((currEnvArchNeeds.statement?.trim().length > 0) &&
+            (currEnvArchNeeds.data_classification_levels?.length > 0))
+    // label
+    let label = "";
+    if(currEnvArchNeeds.data_classification_levels?.length > 0 ) {
+      const classificationLevels = await classificationRequirements
+        .getAllClassificationLevels();
+      label = "Required for ";
+      const selectedLevels = currEnvArchNeeds.data_classification_levels as unknown as string[];
+      selectedLevels.forEach((selectedLabel, index) => {
+        const selectedClasslevelObj = classificationLevels
+          .find(classificationLevel =>
+            classificationLevel.sys_id === selectedLabel) as ClassificationLevelDTO;
+        label = label + selectedClasslevelObj.display
```

Comment:
Can't use `.display` here - use the `buildClassificationLabel` method in `helpers/index.ts` to create labels like `Unclassified/IL2` instead of `Unclassified - IL2`